pkg/lexer/lexer.go

```diff
@@ -107,9 +107,10 @@ func (l *Lexer) readNumber() string {
 }
 
 func (l *Lexer) readLiteral() string {
+	startQuota := l.ch // it's a start string quota
```

Comment:
i believe the word u want to use is "quote" instead of "quota", as in "startQuote"